mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/LegacyMetricOperationsRegistration.java

```diff
@@ -162,8 +162,11 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
         registration.registerOperationHandler(legacyRemoveCustomMetricOperation, legacyRemoveCustomMetricHandler);
     }
 
-    static PathAddress translateProxyPath(OperationContext context) throws OperationFailedException {
-        PathAddress address = context.getCurrentAddress().getParent();
+    private static PathAddress translateProxyPath(OperationContext context) throws OperationFailedException {
+        return translateProxyPath(context, context.getCurrentAddress().getParent());
+    }
+
+    static PathAddress translateProxyPath(OperationContext context, PathAddress address) throws OperationFailedException {
         Set<Resource.ResourceEntry> children = context.readResourceFromRoot(address).getChildren(ProxyConfigurationResourceDefinition.WILDCARD_PATH.getKey());
```

Comment:
This should be context.readResourceFromRoot(address, false).....

This isn't a flaw in this PR but might as well fix it. :)

Reply:
Fixed, thanks for spotting this.